### PR TITLE
Clarify the gameplay rationale for the CBM lore change

### LINKED ISSUE #66632: Why did you remove Earth cybernetics?
### Is your feature request related to a problem? Please describe.

Cybernetics used to be found in vaults in secret XEDRA labs, but now you buy them from Rubik instead. I wanted to know the rationale behind the lore change, but unfortunately can't find any references to it. Was the change made because "the lore did not make sense"? Was it made because bionics were too hard to get? Too random? I have found dozens of places where "Cybernetics were brought to Earth after the cataclysm" is stated as a fact, not none provide any explanation.

### Solution you would like.

After someone in the know responds with a link I will update doc/Lore/technology.md to include the link

### Describe alternatives you have considered.

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by hexagonrecursion**
#### Summary
None

#### Purpose of change
Fixes #66632. I wanted to know why this change to the lore was made, but doc/Lore/technology.md did not tell me. 

#### Describe the solution
Add the missing explanation to the doc/Lore/technology.md

### CODE REVIEW

**Comment by Venera3** [comment]
This does not need to be in the docs.

**Comment by hexagonrecursion** [comment]
And where would you put this info? https://cddawiki.chezzo.com/cdda_wiki/index.php/Bionics ?

File: doc/Lore/technology.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ See [the timeline](./lore-background.md) for lore explanations of the divergent
...
 
 ### Cybernetics
-Cybernetics were not available on CDDA-Earth prior to the portal storms.  Note that this is a recent major change to the lore, and not yet borne out in play.
+Cybernetics were not available on CDDA-Earth prior to the portal storms.  Note that this is a recent major change to the lore, and not yet borne out in play. There are two gameplay reasons for this change. Firstly: too much good loot was concentrated in one "dungeon" - secret labs. [The plan is to make a new different "dungeon" and move cybernetic loot there.](https://www.reddit.com/r/cataclysmdda/comments/wbgtll/comment/iiav21w/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button). Secondly [it was decided to make cybernetic power generation, power storage, low-impact cybernetics and the ability to install cybernetics available without clearing a dangerous location and less dependent on random loot luck](https://www.reddit.com/r/cataclysmdda/comments/t3xum7/comment/hyxhiew/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).
```
Comment (Maleclypse) [inline doc/Lore/technology.md:17]:
```suggestion
Cybernetics were not available on CDDA-Earth prior to the portal storms.  Note that this is a recent major change to the lore. There are two gameplay reasons for this change. Firstly: too much good loot was concentrated in one "dungeon" - secret labs. [The plan is to make a new different "dungeon" and move cybernetic loot there.](https://www.reddit.com/r/cataclysmdda/comments/wbgtll/comment/iiav21w/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button). Secondly [it was decided to make cybernetic power generation, power storage, low-impact cybernetics and the ability to install cybernetics available without clearing a dangerous location and less dependent on random loot luck](https://www.reddit.com/r/cataclysmdda/comments/t3xum7/comment/hyxhiew/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).
```
It has been borne out in play now.

**Comment by Maleclypse** [comment]
I think it's fine to include some gameplay rational to the change mention and then if we want to re-asses including any description of the change at all in another stable or two we should do so.

**Pull request merged**